diff -up a/src/wallet/rpcwallet.cpp b/src/wallet/rpcwallet.cpp
--- a/src/wallet/rpcwallet.cpp	2016-02-17 08:40:03.000000000 +0000
+++ b/src/wallet/rpcwallet.cpp	2016-02-24 07:18:23.881726449 +0000
@@ -6,6 +6,7 @@
 #include "amount.h"
 #include "base58.h"
 #include "chain.h"
+#include "coincontrol.h"
 #include "core_io.h"
 #include "init.h"
 #include "main.h"
@@ -387,9 +388,14 @@
     std::string strError;
     vector<CRecipient> vecSend;
     int nChangePosRet = -1;
+    CCoinControl coinControl;
+    if (mapArgs.count("-destchange")) {
+        CBitcoinAddress addr = CBitcoinAddress(mapArgs["-destchange"]);
+        coinControl.destChange = addr.Get();
+    }
     CRecipient recipient = {scriptPubKey, nValue, fSubtractFeeFromAmount};
     vecSend.push_back(recipient);
-    if (!pwalletMain->CreateTransaction(vecSend, wtxNew, reservekey, nFeeRequired, nChangePosRet, strError)) {
+    if (!pwalletMain->CreateTransaction(vecSend, wtxNew, reservekey, nFeeRequired, nChangePosRet, strError, &coinControl)) {
         if (!fSubtractFeeFromAmount && nValue + nFeeRequired > pwalletMain->GetBalance())
             strError = strprintf("Error: This transaction requires a transaction fee of at least %s because of its amount, complexity, or use of recently received funds!", FormatMoney(nFeeRequired));
         throw JSONRPCError(RPC_WALLET_ERROR, strError);
@@ -1040,7 +1046,12 @@
     CAmount nFeeRequired = 0;
     int nChangePosRet = -1;
     string strFailReason;
-    bool fCreated = pwalletMain->CreateTransaction(vecSend, wtx, keyChange, nFeeRequired, nChangePosRet, strFailReason);
+    CCoinControl coinControl;
+    if (mapArgs.count("-destchange")) {
+        CBitcoinAddress addr = CBitcoinAddress(mapArgs["-destchange"]);
+        coinControl.destChange = addr.Get();
+    }
+    bool fCreated = pwalletMain->CreateTransaction(vecSend, wtx, keyChange, nFeeRequired, nChangePosRet, strFailReason, &coinControl);
     if (!fCreated)
         throw JSONRPCError(RPC_WALLET_INSUFFICIENT_FUNDS, strFailReason);
     if (!pwalletMain->CommitTransaction(wtx, keyChange))
